cl_dll/StudioModelRenderer.h

```diff
@@ -1,4 +1,4 @@
-//========= Copyright � 1996-2002, Valve LLC, All rights reserved. ============
```

Comment:
also get rid of these changes

Reply:
Not sure how to remove that difference from first line of file, change encoding of file not help. Let's it leave it like that.